src/Compilers/Shared/BuildServerConnection.cs

```diff
@@ -13,6 +13,7 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Web;
```

Comment:
This using looks unused.

Reply:
This feedback about unused usings was used.